File: clean-code-practices/practice.py
def find_longest_word_length(text: str) -> int:
    if not text or text.strip() == "":
        return 0
    words = text.strip().split()
    return max(len(w) for w in words)

def reverse_string(text: str) -> str: 
    return text[::-1]

File: clean-code-practices/practice.cs
using System;
using System.Linq;

public class Program
{
    public static int FindLongestWordLength(string text)
    {
        if (string.IsNullOrWhiteSpace(text))
            return 0;

        var words = text.Trim().Split((char[])null, StringSplitOptions.RemoveEmptyEntries);
        return words.Max(w => w.Length);
    }

    public static string ReverseString(string text)
    {
        if (string.IsNullOrEmpty(text)) return text;

        char[] chars = text.ToCharArray();
        Array.Reverse(chars);
        return new string(chars);
    }
}